src/schema/resolvers/user.js

```diff
@@ -8,52 +8,29 @@ export default {
     currentUser: (parent, args, context, info) => context.user
   },
   Mutation: {
-    createUser: async (parent, args, context, info) => {
-      return createUserDB(args.input, context)
-    },
     signup: async (parent, args, context, info) => {
 
-      let newUser = await context.models.user.findOne({ where: { username: args.data.username } })
-      let error = 'user already exists'
-      if (!newUser) {
-        newUser = await createUserDB(args.data, context)
-        error = null
-      }
+      let user = await context.models.user.findOne({ where: { username: args.data.username } })
```

Comment:
Use `const` here instead of `let`. If the variable is not meant to be changed is better to declare it as a constant.